src/lib/controllers/DevicesController.js

```diff
@@ -19,11 +19,40 @@ class DevicesController extends Controller {
     this._deviceRepository = deviceRepository;
   }
 
+  @httpVerb('post')
+  @route('/v1/devices')
+  async claimDevice(postBody: { id: string}): Promise<*> {
+    try {
+      const deviceID = postBody.id;
+      const userID = this.user.id;
+
+      await this._deviceRepository.claimDevice(deviceID, userID);
+
+      return this.ok({ ok: true });
+    } catch (exception) {
+      return this.bad(exception.message);
+    }
+  }
+
+  @httpVerb('delete')
+  @route('/v1/devices/:deviceID')
+  async unclaimDevice(deviceID: string): Promise<*> {
+    try {
+      const userID = this.user.id;
+      await this._deviceRepository.unclaimDevice(deviceID, userID);
+
+      return this.ok({ ok: true });
+    } catch (exception) {
+      return this.bad(exception.message);
```

Comment:
I'm thinking that we should move the `try/catch` to the `RouteConfig` if possible. It seems like all the controllers repeat the same logic over and over.